shm/alcohol.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "cargo.hpp"
+#include <cstddef>
+
+class Alcohol : public Cargo {
+
+  size_t alcoholByVolume_{};
```

Comment:
This will be always `0` because you don't initialize it. Please notice, that you pass this argument in C'tor to `Cargo` instead initialize it here. That's why we always should write tests!